src/CommunityToolkit.Maui/Extensions/ServiceCollectionExtensions.shared.cs

```diff
@@ -35,20 +35,37 @@ public static class ServiceCollectionExtensions
 	}
 
 	/// <summary>
-	/// Adds a <see cref="Popup"/> of the type specified in <typeparamref name="TPopupView"/> and a ViewModel
+	/// Adds a <see cref="View"/> of the type specified in <typeparamref name="TPopupView"/> and a ViewModel
 	/// of the type specified in <typeparamref name="TPopupViewModel"/> to the specified
 	/// <see cref="IServiceCollection"/> with <see cref="ServiceLifetime.Transient"/> lifetime.
 	/// </summary>
-	/// <typeparam name="TPopupView">The type of the Popup to add. Constrained to <see cref="Popup"/></typeparam>
+	/// <typeparam name="TPopupView">The type of the Popup to add. Constrained to <see cref="PopupContainer"/></typeparam>
 	/// <typeparam name="TPopupViewModel">The type of the ViewModel to add. Constrained to 
 	/// <see cref="INotifyPropertyChanged"/></typeparam>
 	/// <param name="services">The <see cref="IServiceCollection"/> to add the service to.</param>
+	/// <param name="lifetime">Service lifetime</param>
 	/// <returns>A reference to this instance after the operation has completed.</returns>
-	public static IServiceCollection AddTransientPopup<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] TPopupView, [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] TPopupViewModel>(this IServiceCollection services)
-		where TPopupView : Popup
-		where TPopupViewModel : INotifyPropertyChanged
+	public static IServiceCollection AddPopup<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] TPopupView, [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] TPopupViewModel>(this IServiceCollection services, ServiceLifetime lifetime = ServiceLifetime.Transient)
+		where TPopupView : View
```

Comment:
`PopupService` uses `IView` for consistency would make sense to use the interface here?

Reply:
Yup - Makes sense 
```suggestion
		where TPopupView : IView
```